corehq/messaging/scheduling/models/abstract.py

```diff
@@ -33,6 +38,10 @@ class Schedule(models.Model):
     # This metadata will be passed to any messages generated from this schedule.
     custom_metadata = jsonfield.JSONField(null=True, default=None)
 
+    # One of the UI_TYPE_* constants describing the type of UI that should be used
+    # to edit this schedule.
+    ui_type = models.CharField(max_length=1, default='X')
```

Comment:
Would it be worth making a constant like `UI_TYPE_UNKNOWN` for clarity?

Reply:
agreed, https://github.com/dimagi/commcare-hq/pull/18495/commits/d2f4ffe91577150a1d4458d5685a1c0fc844b278